picard/ui/mainwindow.py

```diff
@@ -674,7 +679,13 @@ def search(self):
         """Search for album, artist or track on the MusicBrainz website."""
         text = self.search_edit.text()
         type = self.search_combo.itemData(self.search_combo.currentIndex())
-        self.tagger.search(text, type, config.setting["use_adv_search_syntax"])
+        if config.setting["builtin_search"]:
+            if type == "track":
```

Comment:
Please ignore this for now. I'll open two more PRs just after this one's merged, for album and artist dialog. All types would be covered after those. (Should've mentioned this in a comment.)